src/Pages/ShopPage/ShopComponents/ProductCard/ProductCard.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useState } from 'react';
+import { Link } from "react-router-dom"
+import './ProductCard.css';
+
+const ProductCard = (props) => {
+    const [toggleLike, setToggleLike] = useState(false);
+
+    const { _id, title, originalprice, currentprice, discount, reviews, rating, imgSrc, inStock, isFav, altTxt } = props.productdetails;
+
+    return (
+
+        <div className={inStock ? "card ver-card no-bg-color" : "card ver-card no-bg-color overlay"} >
+            <div className="img-container pos-rel">
+                <Link to="/singleprod">
+                    <img className="img-resp" src={imgSrc} alt={altTxt} />
+                </Link>
+                <div className="rating-row bdg pos-abs bottom-left">
+                    <i className="badge-star fas fa-star"></i>
+                    <p className="xsm-title">{rating}</p>
+                    <i className="badge-circle fas fa-circle"></i>
+                    <p className="review-num xsm-p grey-txt">({reviews})</p>
+                </div>
```

Comment:
Appreciation: I liked you added item rating on card itself, reminds me on Myntra 😄

Reply:
Thanks a lot and yes it is inspired from myntra xD